### PR TITLE
relationalai 0.3.4 :snowflake:

### PR DESCRIPTION
**Pull request by lorepirri**
relationalai 0.3.4 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-4599]
- dev_url: :warning: not available, see comments in the ticket
- conda-forge: <feedstock not found on conda-forge>
- pypi: https://pypi.org/project/relationalai/0.3.4
- pypi inspector: https://inspector.pypi.io/project/relationalai/0.3.4

### Explanation of changes:

- new feedstock, patch to remove `debugger` CLI command.
- ⚠️ ~license not known~ LICENSE in the `recipe/` folder, `Apache-2.0`. See comment https://anaconda.atlassian.net/browse/PKG-4599?focusedCommentId=218479
- a repository is not available yet, no `home`, no `dev_url`, no `doc_url`.

[PKG-4599]: https://anaconda.atlassian.net/browse/PKG-4599?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    - rai debugger
+    # We run only this one test, because all the other will require credentials
+    # for the snowflake/azure backend, and need to connect to their services.
+    #
+    # Skip test_new_profile because the snapshot is similar but not exactly
+    # matching the computed value (same fields, different order).
+    - pytest -vv tests/unit -k "not test_new_profile"
+
+about:
+  summary: The Python library for building and querying knowledge graphs with RelationalAI.
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:83]:
```suggestion
  home: https://relational.ai/
  doc_url: https://relational.ai/docs/getting_started
  # The repository is at the moment private.
  # dev_url: https://github.com/RelationalAI/relationalai-python
  summary: The Python library for building and querying knowledge graphs with RelationalAI.
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  license: Apache-2.0
+  license_file: LICENSE
+  license_family: Apache
+  # 
+  # 
+  # The repository is at the moment private.
+  #
+  # home:
+  # doc_url:
+  # dev_url: https://github.com/RelationalAI/relationalai-python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    #
+    # Skip test_new_profile because the snapshot is similar but not exactly
+    # matching the computed value (same fields, different order).
+    - pytest -vv tests/unit -k "not test_new_profile"
+
+about:
+  summary: The Python library for building and querying knowledge graphs with RelationalAI.
+  description: |
+    The Python library for building and querying knowledge graphs with RelationalAI.
+  license: Apache-2.0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:86]: What is the source of the license? It's not mentioned in the source code

**Comment by skupr-anaconda** [comment]
@lorepirri As you build **v0.3.4** I updated the PR's title and links

File: recipe/meta.yaml (hunk 3 above)
Comment (lorepirri) [reply recipe/meta.yaml:86]: Internal. See comment https://anaconda.atlassian.net/browse/PKG-4599?focusedCommentId=218479

**Comment by lorepirri** [comment]
> @lorepirri As you build **v0.3.4** I updated the PR's title and links

Ah thank you, I forgot about that 👍

**Review by skupr-anaconda — APPROVED** [review]

**Review by ViridianMelody — APPROVED** [review]

**Pull request merged**